RecoTracker/Configuration/python/customizeMinPtForHitRecoveryInGluedDet.py

```diff
@@ -13,4 +13,14 @@ def customizeHitRecoveryInGluedDetOn(process):
        process.Chi2MeasurementEstimatorForP5.MinPtForHitRecoveryInGluedDet = 100000
    return process
 
+def customizeHitRecoveryInGluedDetTkSeedsOnly(process):
+   process = customizeMinPtForHitRecoveryInGluedDet(process,0.9)
+   if hasattr(process, "Chi2MeasurementEstimatorForP5"): # keep disabled for cosmics
+       process.Chi2MeasurementEstimatorForP5.MinPtForHitRecoveryInGluedDet = 100000
+   for X in "muonSeededMeasurementEstimatorForOutIn", "hitCollectorForOutInMuonSeeds":
```

Comment:
looking at this closer, comparing to #17807 ,
there we have muonSeededMeasurementEstimatorForInOutBase and muonSeededMeasurementEstimatorForOutInBase while this one has it only for OutIn.
Should the muonSeededMeasurementEstimatorForOutIn be modified as well?